src/CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 ## [Unreleased]
 ### Added
-* Added `math.square()` to compute the square (x * x).
-* Added `math.orthonormal_basis()` to compute an orthonormal basis from a single unit length vector.
-* Added `math.sign(x)` for int, int2, int3 and int4.
+* Added `math.square` to compute the square (x * x).
+* Added `math.orthonormal_basis` to compute an orthonormal basis from a single unit length vector.
+* Added `math.sign` for int, int2, int3 and int4.
+* Added `math.chgsign` for float, float2, float3, and float4.
+* Added `math.Euler` to convert a quaternion to Euler angles.
+* Added `quaternion.angle` to compute the angle between two unit quaternions.
```

Comment:
@unpacklo actually the quaternion angle method is in `math.angle(quaternion, quaternion)`.
Do we want to move it ?

Reply:
We can leave it in the math class, I'll update the changelog.